ranges.md

```diff
@@ -0,0 +1,384 @@
+       included by a SemVer Comparator Set unless they are included by all
+       of the Comparators in the Comparator Set, and one or more of the
+       following conditions are met:
+
+        1. The implementation has provided the user with an option to
+           explicitly include PRERELEASE versions, and the user has set this
+           option.  This SHOULD NOT be the default behavior when resolving
+           software dependencies.
+
+        1. One or more of the Comparators in the Comparator Set have the
+           same MAJOR, MINOR, and PATCH version as the SemVer Version
+           string being considered, and has a PRERELEASE version, and would
+           normally include the SemVer Version string by virtue of
+           precedence comparison.
+
+            For example, the Range `>=1.0.0-alpha` would include the
+            Version `1.0.0-beta` but _not_ the Version `1.0.1-beta`.
+
+    1. SemVer Version strings without a PRERELEASE version are included in
+       a SemVer Comparator Set if they are included by all of the
+       Comparators in the Comparator Set.
+
+1. `Advanced Comparators` are defined in terms of their reduction to a set
+   of primitive Comparators.
+
+    1. The empty string, asterisk `*`, capital `X` and lowercase `x` are
+       equivalent to `>=0.0.0`, or `>=0.0.0-0` if PRERELEASE versions are
+       being included.
+
+    1. A "tilde version" of the form `~I.J.K` will match any version with a
+       MAJOR version equal to `I`, a MINOR version equal to `J`, and a
+       PATCH version equal to or greater than `K`.  These are equivalent to
+       `>=I.J.K <I.J.(K+1)` or `>=I.J.K <I.J.(K+1)-0` if PRERELEASE
+       versions are being included.
+
+    1. A "caret version" of the form `^I.J.K` allows changes that do not
+       increment the first non-zero portion of the SemVer
+       (MAJOR,MINOR,PATCH) tuple.
+
+        1. `^0.0.K` will match any version `>=0.0.K <0.0.(K+1)-0`.  Note that
+           this is equivalent to `=0.0.K` if PRERELEASE versions are not
+           included.
+
+        1. `^0.J.K` will match any version in the range `>=0.J.K
+           <0.J.(K+1)-0`.
+
+        1. `^I.J.K` will match any version in the range `>=I.J.K
+           <I.(J+1).0-0`.
+
+    1. A "hyphen range" expresses an inclusive range of versions.  `I.J.K -
+       L.M.N` is equivalent to `>=I.J.K <=L.M.N`
+
+    1. Partial versions are versions of the form, `I` or `I.J` rather than
+       `I.J.K`, and behave the following impacts when used in SemVer
+       Ranges.
+
+        1. A partial version with the `=` operator, or no operator, will
+           match any version whose parts of the SemVer (MAJOR,MINOR,PATCH)
+           tuple are equivalent to the specified portions in the partial
+           version.  Thus:
+
+            1. `I.J` or `=I.J` are equivalent to `>=I.J.0 <I.(J+1).0-0`, or
+               `>=I.J.0-0 <I.(J+1).0-0` if PRERELEASE versions are being
+               included.
```

Comment:
I believe this is inconsistent with node-semver's implementation of `=I.J`. In particular:

```console
> semver.satisfies('1.0.0-beta', '>1.0.0-alpha =1.0')
false
```

Under my interpretation of the spec as currently written, I would expect `true`. PRERELEASE versions of 1.0.0 **are** being included as per the folllowing rule, because the comparator `>1.0.0-alpha` has the same MAJOR, MINOR, and PATCH version as `1.0.0-beta`, and has a PRERELEASE version, and would normally include `1.0.0-beta` by virtue of precedence comparison:

https://github.com/semver/semver/blob/efcff2c838c9945f79bfd21c1df0073271bcc29c/ranges.md#L128-L131
https://github.com/semver/semver/blob/efcff2c838c9945f79bfd21c1df0073271bcc29c/ranges.md#L138-L142

Therefore the interpretation of `=1.0` as `>=1.0.0-0 <1.1.0-0` is the one that applies, and `1.0.0-beta` is supposed to match that.

Is the intended interpretation of `=I.J` the one in node-semver or the one in the spec?